packages/js/src/ui/components/Inbox.tsx

```diff
@@ -11,6 +16,8 @@ export type InboxProps = {
   tabs?: Array<{ label: string; value: Array<string> }>;
   mountNotification?: NotificationMounter;
   mountBell?: BellMounter;
+  onNotificationClick?: NotificationClickHandler;
+  onActionClick?: NotificationActionClickHandler;
```

Comment:
two new props that could be passed on to the `Inbox` or `Notifications` component